Bank#open throws exception

```
java.lang.BootstrapMethodError: call site initialization exception
    at java.lang.invoke.CallSite.makeSite(CallSite.java:341)
    at java.lang.invoke.MethodHandleNatives.linkCallSiteImpl(MethodHandleNatives.java:307)
    at java.lang.invoke.MethodHandleNatives.linkCallSite(MethodHandleNatives.java:297)
    at org.powerbot.script.rt4.BasicQuery.viewable(BasicQuery.java:185)
    at org.powerbot.script.rt4.Bank.getBank(Bank.java:40)
    at org.powerbot.script.rt4.Bank.open(Bank.java:107)
    at scripts.FishingInAlkharid.TaskBankingDepositFish.execute(TaskBankingDepositFish.java:56)
    at scripts.FishingInAlkharid.ScriptFishingInAlkharid.poll(ScriptFishingInAlkharid.java:129)
    at org.powerbot.script.PollingScript$1.run(PollingScript.java:49)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
    at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.invoke.LambdaConversionException: Invalid receiver type interface org.powerbot.script.Locatable; not a subtype of implementation type interface org.powerbot.script.Viewable
    at java.lang.invoke.AbstractValidatingLambdaMetafactory.validateMetafactoryArgs(AbstractValidatingLambdaMetafactory.java:233)
    at java.lang.invoke.LambdaMetafactory.metafactory(LambdaMetafactory.java:303)
    at java.lang.invoke.CallSite.makeSite(CallSite.java:302)
    ... 11 more
```

--- a/src/main/java/org/powerbot/script/rt4/BasicQuery.java
+++ b/src/main/java/org/powerbot/script/rt4/BasicQuery.java
@@ -182,6 +182,7 @@ public abstract class BasicQuery<K extends Locatable & Identifiable & Nameable &
 	 */
 	@Override
 	public BasicQuery<K> viewable() {
-		return select(Viewable::inViewport);
+		//DO NOT REPLACE WITH METHOD REFERENCE
+		return select(k -> k.inViewport());
 	}
 }

--- a/src/main/java/org/powerbot/script/rt4/BasicQuery.java
+++ b/src/main/java/org/powerbot/script/rt4/BasicQuery.java
@@ -183,6 +183,7 @@ public abstract class BasicQuery<K extends Locatable & Identifiable & Nameable &
 	@Override
 	public BasicQuery<K> viewable() {
 		//DO NOT REPLACE WITH METHOD REFERENCE
+		//see #2119
 		return select(k -> k.inViewport());
 	}
 }

